mappings/net/minecraft/block/BlockSugarCane.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS bke net/minecraft/block/BlockSugarCane
+	FIELD a age Lboo;
```

Comment:
I'll have an automated mechanism for this ,but if you want to keep it, static final fields are generally UPPERCASE_WITH_SNAKES.